docs/source/conventions/computed_identifiers.rst

```diff
@@ -63,29 +63,22 @@ Implementations MUST adhere to the following requirements:
 
 * Implementations MUST use the normalization, serialization, and
   digest mechanisms described in this section when generating GA4GH
-  Computed Identifiers.  Implementations MUST NOT use any other
+  Computed Identifiers. Implementations MUST NOT use any other
   normalization, serialization, or digest mechanism to generate a
   GA4GH Computed Identifier.
 
-* When computing identifiers, implementations MUST ensure that each 
-  nested :ref:`Ga4ghIdentifiableObject` is referenced with a GA4GH 
+* When computing identifiers, implementations MUST ensure that each
+  nested :ref:`Ga4ghIdentifiableObject` is referenced with a GA4GH
   Computed Identifier.
 
-.. note:: The GA4GH schema MAY be used with identifiers from any
-          namespace. For example, a SequenceLocation may be defined
-          using a `sequence_id` = ``refseq:NC_000019.10``.  However,
-          an implementation of the Computed Identifier algorithm MUST
-          first translate sequence accessions to GA4GH RefGet ``SQ``
-          accessions to be compliant with this specification.
-
```

Comment:
I just removed the entire section. Did we want to say anything in particular about RefGet accessions being required now?

Reply:
Good call, agree.